packages/aws-cdk/README.md

```diff
@@ -746,6 +746,32 @@ NOTICES
 If you don’t want to see a notice anymore, use "cdk acknowledge <id>". For example, "cdk acknowledge 16603".
 ```
 
+### `cdk migrate`
+
+⚠️**CAUTION**⚠️ CDK migrate is currently experimental. We make no guarantees about the outcome or stability of the Command or it's output.
+
+Generates a CDK application from an existing Cloudformation template in any of the CDK supported languages. 
+Required Arguments:
+* `--from-path <my_file_path>` - Takes the relative filse path to the JSON or YAML template of your choice
+* `--stack-name <my_stack_name>` - Used to name both the CDK application and stack. 
+Optional Arguments:
+* `--output-path <my_output_path>` - file path to where the file should be generated. 
+  Default is the current working directory
```

Comment:
file? What type of file? Initially I thought this was supposed to be the generated CDK app, which is a folder (or maybe a zip file), but now I'm thinking it means the stack.ts file. If so it should say something like:

```
* `--output-path <my_output_path>` - path where the generated CDK Stack will be located.
  Default is `./foo.ts`
```

Reply:
unclear by me. Fixing